Source/WebCore/display/css/DisplayStyle.cpp

```diff
@@ -91,12 +91,12 @@ Style::Style(const RenderStyle& style, const RenderStyle* styleForBackground)
     , m_tabSize(style.tabSize())
     , m_opacity(style.opacity())
 {
-    // FIXME: Is currentColor resolved here?
     m_color = style.visitedDependentColorWithColorFilter(CSSPropertyColor);
 
     if (styleForBackground)
         setupBackground(*styleForBackground);
 
+    // FIXME: m_boxShadow should use a custom data structure with a resolved color (aka Color).
```

Comment:
Could we have a bugzilla bug filed for this and referenced here?

Reply:
Added